torchx/schedulers/ray_scheduler.py

```diff
@@ -132,7 +155,65 @@ def run_opts(self) -> runopts:
         return opts
 
     def schedule(self, dryrun_info: AppDryRunInfo[RayJob]) -> str:
-        raise NotImplementedError()
+        cfg: RayJob = dryrun_info.request
+
+        # not sure what to do with this
+        verbose: bool = False
+        appId: str = ""
+        cluster_name: Optional[str] = None
+
+        # Create serialized actors for ray_driver.py
+        actors = cfg.actors
+
+        dirpath = mkdtemp()
+
+        serialize(actors, dirpath)
+        
+        if cfg.cluster_config_file:
+            ip_address = ray_autoscaler_sdk.get_head_node_ip(cfg.cluster_config_file)
+
+        elif cfg.cluster_address:
+            ip_address = cfg.cluster_address
+
+        else:
+            ip_address = "127.0.0.1"
```

Comment:
If you define a default value as part of `runopts`, you won't need this last statement.